flink-libraries/flink-table/src/test/java/org/apache/flink/api/java/batch/sql/SqlITCase.java

```diff
@@ -118,4 +119,39 @@ public void testJoin() throws Exception {
 		String expected = "Hi,Hallo\n" + "Hello,Hallo Welt\n" + "Hello world,Hallo Welt\n";
 		compareResultAsText(results, expected);
 	}
+
+
+	@Test
+	public void testLikeWithEscapeFromDataSet() throws Exception {
```

Comment:
We try to keep the number of integration tests for the Table API to a minimum to not exceed Travis' build time threshold of 2 hours. I think we do not need the  IT cases, the `ScalarFunctionsTest` should cover all cases. 

You can add new tests to `ScalarFunctionsTest` called `testLikeWithEscape`, `testSimilarWithEscape` just like what `testLike` did.